frontend/public/components/app.jsx

```diff
@@ -286,12 +315,7 @@ graphQLReady.onReady(() => {
       <Provider store={store}>
         <Router history={history} basename={window.SERVER_FLAGS.basePath}>
           <Switch>
-            <Route
-              path="/k8s/ns/:ns/virtualmachineinstances/:name/standaloneconsole"
-              render={(componentProps) => (
-                <AsyncComponent loader={consoleLoader} {...componentProps} />
-              )}
-            />
+            <Route path="/standalone" component={StandaloneRouter} />
```

Comment:
Why force the route to start with `standalone` ?

Reply:
@christianvogt Scoping under `/standalone` is a convention for all pages where we don't want the common Console page layout. Other plugin-contributed route pages will be processed by the default fallback

```ts
<Route path="/" component={AppWithTranslation} />
```

Reply:
I see the fallback. However I don't see the reasoning behind the `/standalone` prefix.
You can see how we contribute the `/terminal` today. I'd like to use this extension to do this. However I don't want `/standalone/terminal` as the URL.

Reply:
If we allow any path as standalone, without prefix, then anytime you open the path and plugin is not resolved yet, you will fall through to App component (`/` path) which will render Masthead, Navigation and other components. After some time, plugin gets resolved, the `App` component gets unmounted and plugin's component gets rendered.

Thats why I opted for predefined `standalone` prefix, where we can tell this is a standalone page and we need to wait for plugins to be resolved if the path is not matched yet.

Reply:
You should not be using `useResolvedExtensions` here but instead `useExtensions` because we don't want to pre-resolve all standalone pages upfront.
The declarative extensions will be available immediately without the need for loading anything extra and therefore can setup the routes immediately. The use case you describe should not happen.